frontend/ui/widget/bookmapwidget.lua

```diff
@@ -1200,16 +1207,6 @@ function BookMapWidget:showMenu()
                 self:showGestures()
             end,
         }},
-        {{
-            text = _("Page browser on tap"),
-            checked_func = function()
```

Comment:
I'd rather you to let this be here - so it's obvious how the menu is laid out on Touch devices, and we don't have to guess.
You can just table.remove() it in a `if not Device:isTouchDevice() then` in the place you were adding it.

Reply:
done and done